packages/dd-trace/src/debugger/devtools_client/send.js

```diff
@@ -37,5 +39,13 @@ function send (message, logger, snapshot, cb) {
     'debugger.snapshot': snapshot
   }
 
-  request(JSON.stringify(payload), opts, cb)
+  let json = JSON.stringify(payload)
+
+  if (Buffer.byteLength(json) > MAX_PAYLOAD_SIZE) {
+    // TODO: This is a very crude way to handle large payloads. Proper pruning will be implemented later (DEBUG-2624)
+    delete payload['debugger.snapshot']
```

Comment:
kinda harsh :D 

remove 'debugger.snapshot' would cause probe-tracker to mark the probe as error: as the probe would trigger emitting start but there are not events marked with the probe-id and would default to error state.  

I would suggest remove 'debugger.snapshot.captures' or replace it with notCaptureReasons: 'snapshot was too large' - at least give the user some feedback on why the don't see the snapshot.

Reply:
Oh, good point. My intent was to only rmeove the `captures` part, but the JSON schema threw me for a loop since we're collecting a snapshot, the property is called `snapshot` and it's called "snapshot pruning" 🙄 I'll push a fix 👍

Reply:
Fixed 👍